components/sharedfiles.js

```diff
@@ -20,14 +20,25 @@ SteamCommunity.prototype.deleteSharedFileComment = function(userID, sharedFileId
 		"form": {
 			"gidcomment": cid,
 			"count": 10,
+			"json": 1,
 			"sessionid": this.getSessionID()
-		}
+		},
+		"json": true
 	}, function(err, response, body) {
 		if (!callback) {
 			return;
 		}
 
-		callback(err);
+		if (err) {
+			callback(err);
+			return;
+		}
+
+		if (body.success) {
+			callback(null);
+		} else {
+			callback(new Error(body.error));
+		}
```

Comment:
Is this tested and confirmed working? Typically, when Steam returns a `success` key, it's an eresult and not a bool.

Reply:
Yep, Steam returns the following here: `{ success: false, error: 'There was a problem deleting this comment. Please try again.' }`